pkg/actions/pkg_list.go

```diff
@@ -109,3 +112,12 @@ func (pl *PkgList) Run() error {
 	t.AppendBulk(rows)
 	return t.Render()
 }
+
+func (pl *PkgList) addRow(regName, libName string, isInstalled bool) []string {
+	row := []string{regName, libName}
+	if isInstalled {
+		row = append(row, pkgInstalled)
```

Comment:
Do we really need the star if we are only listing installed things?

Reply:
I wanted to keep the column headings the same.